Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3-slim
```

Comment:
In that case, feel free to ignore my nonsense here and maybe we bump from Python 2.7 to Python 3.x in the .travis.yml file (so they're both in sync).